pilot/pkg/model/push_context.go

```diff
@@ -93,7 +93,6 @@ type PushContext struct {
 	// the dest rule based on the most specific host match, and not just any destination rule
 	namespaceLocalDestRules    map[string]*processedDestRules
 	namespaceExportedDestRules map[string]*processedDestRules
-	allExportedDestRules       *processedDestRules
```

Comment:
Can you also remove the comment above this line?

Reply:
done